phaseshifts/atorb.py

```
@@ -858,3 +1255,358 @@ def calculate_Q_density(**kwargs):
+        successful = False
+        try:
+            kwargs = kwargs.pop("fmt") if "fmt" in kwargs else kwargs
+            # generate buffer string of input for each element
+            for element in set(elements):
+                Atorb.gen_input(element, atorb_file=io, fmt="eeasisss", **kwargs)
+            # write buffered string to disk
+            with open(atorb_file, "w") as f:
+                f.write(io.getvalue())
+            successful = True
+        except Exception as e:
+            raise e
```

Ruff (0.14.10)

Remove exception handler; error is immediately re-raised

(TRY203)